website/docs/docs/cloud-integrations/configure-auto-exposures.md

```diff
@@ -30,12 +30,12 @@ To access the features, you should meet the following:
 
 This section of the document explains the steps you need to set up the auto-exposures integration with Tableau. Once you've set this up in Tableau and dbt Cloud, you can view the [auto-exposures](/docs/collaborate/auto-exposures#view-auto-exposures-in-dbt-explorer) in dbt Explorer.
 
-To set up [personal access tokens (PATs)](/docs/dbt-cloud-apis/user-tokens#using-the-new-personal-access-tokens) needed for auto exposures, ask a site admin to configure it for the account.
+To set up [personal access tokens (PATs)](https://help.tableau.com/current/server/en-us/security_personal_access_tokens.htm) needed for auto exposures, ask a site admin to configure it for the account.
 
 1. Ensure you or a site admin enables PATs for the account in Tableau.
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/tableau-enable-pat.jpg" title="Enable PATs for the account in Tableau"/>
 
-2. Create a PAT that you can add to dbt Cloud to pull in Tableau metadata for auto exposures.
+2. Create a PAT that you can add to dbt Cloud to pull in Tableau metadata for auto exposures. Ensure the user creating the PAT has access to collections/folders, as the PAT only grants access matching the creator's existing privileges.
```

Comment:
```suggestion
2. Create a PAT that you can add to dbt Cloud to pull in Tableau metadata for auto exposures. The user creating the PAT will need to make sure they have access to collections/folders, as the PAT only grants access matching the creator's existing privileges.
```

Reply:
Hiya @mirnawong1 

I left this non-blocking suggestion - changed the sentence slightly to avoid the repetition of "ensure".

Kind Regards
Natalie

Reply:
thanks so much @nataliefiann ! i'm going to keep it as it bc i think it's a bit more concise and uses the imperative, which is usually more effective.